Automattic/wp-calypso
# Framework: removed sites-list usage from wpcom-view/views/embed

I’m not proud of the solution found to remove sites list here. I had to pass the redux store to EventEmitters of the views, so they can subscribe and maintain their logic. Doing this was the way I found to avoid huge refactors. If anyone sees a better approach to remove sites list here, please post it and then I will implement it.

To test:
Open the tinymce post editor add a link there that renders an Embed views (just copy paste the link in the editor as is), e.g: youtube video, slideshare slides, etc… Check that the preview of the content renders well inside the editor.
